src/sentry/receivers/onboarding.py

```diff
@@ -388,6 +360,32 @@ def record_user_context_received(project, event, **kwargs):
 
 
 event_processed.connect(record_user_context_received, weak=False)
+
+
+@first_event_with_minified_stack_trace_received.connect(weak=False)
+def record_event_with_first_minified_stack_trace_for_project(project, event, **kwargs):
+    try:
+        user: APIUser = Organization.objects.get(id=project.organization_id).get_default_owner()
+    except IndexError:
+        logging.getLogger("sentry").warning(
```

Comment:
@priscilawebdev can you do this at the top of the fil?
```
logger = logging.getLogger(__name__)
```
We want the logger name to match the file